.github/workflows/crds-verify-kind.yaml

```diff
@@ -52,7 +52,6 @@ jobs:
       matrix:
         # Latest k8s versions. There's no series-based tag, nor is there a latest tag.
         k8s:
-          - 1.15.12
```

Comment:
Please keep the v1.15 version here.

Reply:
IIRC, the k8s 1.15 does not support CRD v1 version.